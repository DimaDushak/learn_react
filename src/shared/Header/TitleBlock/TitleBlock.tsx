import React from 'react';
import styles from './titleblock.css';

export default function TitleBlock(): JSX.Element {
    return <div className={styles.titleBlock}>title</div>;
}
