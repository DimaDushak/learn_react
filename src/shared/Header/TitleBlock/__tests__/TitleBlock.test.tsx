import { render } from '@testing-library/react';
import React from 'react';
import TitleBlock from '../TitleBlock';

test('should render TitleBlock (snapshot)', () => {
    const { container } = render(<TitleBlock />);

    expect(container.firstChild).toMatchSnapshot();
});
