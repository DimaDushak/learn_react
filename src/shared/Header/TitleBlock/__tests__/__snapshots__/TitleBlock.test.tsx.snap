// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render TitleBlock (snapshot) 1`] = `
<div
  class="titleBlock"
>
  title
</div>
`;
